mozci/push.py

```diff
@@ -1094,7 +1107,25 @@ def classify(
         - good push: when there are only intermittent failures
         - unknown state: when other tasks are failing
         """
-        regressions = self.classify_regressions(confidence_medium, confidence_high)
+        cache_path = f"{self.push_uuid}/classify_group_tasks/classified_regressions"
```

Comment:
Maybe we should avoid to cache the classify results completely:
- when we run the command to analyze pushes for debugging, we don't want the cache, as we want to experiment with different values and variations of the algorithm;
- when we run the command to analyze pushes at "real time", the cache is not useful because the push is definitely not finalized.

So in the end we'd basically never benefit from this specific caching.

Reply:
Ok, so I should remove it then?

Reply:
Yes, thanks